pkg/controller/plan/adapter/vsphere/builder.go

```diff
@@ -209,9 +215,32 @@ func (r *Builder) PodEnvironment(vmRef ref.Ref, sourceSecret *core.Secret) (env
 			Value: fingerprint,
 		},
 	)
+	if macsToIps != "" {
+		env = append(env, core.EnvVar{
+			Name:  "V2V_staticIPs",
+			Value: macsToIps,
+		})
+	}
 	return
 }
 
+func (r *Builder) mapMacStaticIps(vm *model.VM) string {
+	if !isWindows(vm) {
+		return ""
+	}
+	configurations := []string{}
+	for _, guestNetwork := range vm.GuestNetworks {
+		if guestNetwork.Origin == string(types.NetIpConfigInfoIpAddressOriginManual) {
+			configurations = append(configurations, fmt.Sprintf("%s:ip:%s", guestNetwork.MAC, guestNetwork.IP))
+		}
+	}
+	return strings.Join(configurations, "_")
+}
+
+func isWindows(vm *model.VM) bool {
+	return vm.GuestID != "" && strings.Contains(vm.GuestID, WindowsPrefix) || strings.Contains(vm.GuestName, WindowsPrefix)
```

Comment:
seems we can simplify this even further with
```suggestion
	return strings.Contains(vm.GuestID, WindowsPrefix) || strings.Contains(vm.GuestName, WindowsPrefix)
```
right?